src/Momento.Sdk/Internal/ControlGrpcManager.cs

```diff
@@ -1,9 +1,18 @@
-﻿#pragma warning disable 1591
+﻿#if NETFRAMEWORK
```

Comment:
under what conditions does this check succeed?  Would it be worth adding a comment here that describes the conditions?